supabase/functions/register-team-email-trigger/README.md

```diff
@@ -0,0 +1,12 @@
+## Setting up the register team email trigger
+
+This function is triggered when a new team is created and sends an email to a specified email list.
+
```

Comment:
You should note that the `EMAILS` environment variable should also be set to the emails it needs to send an email to